packages/core/config/utils.js

```diff
@@ -0,0 +1,18 @@
+const {getPaths} = require('@redwoodjs/internal')
+const {existsSync} = require('fs')
+const merge = require('webpack-merge')
+
+module.exports['mergeUserWebpackConfig'] = (env, baseConfig) => {
```

Comment:
Could you rename `env` to `mode` since `env` could be `process.env`

Reply:
Renamed